src/components/date-picker/DatePicker.scss

```diff
@@ -19,8 +19,8 @@
     }
 
     .date-picker-description {
-        display: block;
         color: $bdl-gray-62;
+        font-style: italic;
```

Comment:
Does this have to be italic? Italic doesn't work so well with Asian characters and fonts....

Reply:
it was originally a \<i\> tag, so this was just to keep default behavior.  This can be modified by overriding the styling during usage

Reply:
@ehoogerbeets checking with design

Reply:
per offline conversations, design input is to remove italics